policy/modules/system/fstools.te

```diff
@@ -145,6 +150,8 @@ mls_file_write_all_levels(fsadm_t)
 
 selinux_getattr_fs(fsadm_t)
 
+storage_dev_filetrans_fixed_disk_control(fsadm_t, "megaraid_sas_ioctl_node")
```

Comment:
I'd rather drop the name unless there is a conflicting type_transition that I'm not seeing.

Reply:
ok

Reply:
libsepol.expand_terule_helper: conflicting TE rule for (kernel_t, device_t:chr_file):  old was event_device_t, new is fixed_disk_device_t
libsepol.expand_module: Error during expand

Seems to be a conflicting transition

Reply:
dev_filetrans_input_dev(kernel_t)
storage_dev_filetrans_fixed_disk(kernel_t)

The existence of the above 2 lines precludes having storage_dev_filetrans_fixed_disk() work on both chr_file and blk_file. Shall I replace storage_dev_filetrans_fixed_disk with 2 new interfaces and if so what shall I name them?

Reply:
In that case, I propose: try adding `chr_file` to `storage_dev_filetrans_fixed_disk()` and then keep the `"megaraid_sas_ioctl_node"` in the call.

Reply:
ok